### PR TITLE
add peers example

### PR DESCRIPTION
**Pull request by bernardoaraujor**

### CODE REVIEW

**Comment by bernardoaraujor** [comment]
@rufsam I think we need to fix something on `bee`.

from @Thoralf-M 's words:

> we are using the types from bee https://github.com/iotaledger/bee/blob/337d74652cf25468cc02ce9c0eedc69d19709678/bee-api/bee-rest-api/src/types.rs#L607
> And the gossip field is also here https://editor.swagger.io/?url=https://raw.githubusercontent.com/rufsam/protocol-rfcs/master/text/0026-rest-api/rest-api.yaml
> So looks like it just wasn't added it hornet yet

File: examples/peers.rs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+// SPDX-License-Identifier: Apache-2.0
+
+//! cargo run --example nodeinfo --release
+use iota::Client;
+
+/// In this example we get the nodeinfo
+#[tokio::main]
+async fn main() {
+    let iota = Client::builder() // Crate a client instance builder
+        .with_network("alphanet1")
```
Comment (Thoralf-M) [inline examples/peers.rs]: Please remove with_network, it's not needed and could cause problems

Comment (bernardoaraujor) [reply examples/peers.rs]: you mean only line 11, correct?

Comment (Thoralf-M) [reply examples/peers.rs]: Yes

**Review by Thoralf-M — APPROVED** [review]

File: examples/peers.rs [hunk 2]
```diff
@@ -0,0 +1,19 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! cargo run --example nodeinfo --release
```
Comment (Thoralf-M) [inline examples/peers.rs]: Oh and update this to `//! cargo run --example peers --release`

**Pull request merged**